Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: postgresql
+  repository: https://jhvhs.github.io/concourse-chart
```

Comment:
Not sure if you plan on maintaining that chart repo indefinitely, might be a better idea to continue depending on the bitnami chart: https://hub.helm.sh/charts/bitnami/postgresql

```suggestion
  repository: https://charts.bitnami.com
```

Reply:
We have opened a pull request with similar changes to the stable/postgresql chart. Using the current version of it would defeat the purpose of this PR. In the meantime we'll maintain the chart above.